### ISSUE TITLE
Compilation errors in 0.4.4

### ISSUE THREAD

**Issue by kornelski** [kaj/ructe#37]
error[E0412]: cannot find type `ErrorKind` in this scope
> ructe-0.4.4/src/errors.rs:8:29
    |
8   |             static ref ERR: ErrorKind = def_error($msg);
    |                             ^^^^^^^^^ not found in this scope

> ructe-0.4.4/src/templateexpression.rs:199:26
    |
199 |            return_error!(err_str!("Expected \"{\""), char!('{')),
    |                          -------------------------- in this macro invocation
help: possible candidates are found in other modules, you can import them into scope
    |
1   | use nom::ErrorKind;
    |
1   | use std::io::ErrorKind;
    |

>  no method named `clone` found for type `templateexpression::template_block::ERR` in the current scope
   --> ructe-0.4.4/src/errors.rs:10:13
    |
10  |         ERR.clone()
    |             ^^^^^
    |

> ructe-0.4.4/src/templateexpression.rs:199:26
    |
199 |            return_error!(err_str!("Expected \"{\""), char!('{')),
    |                          --------------------------
    |                          |
    |                          method `clone` not found for this
    |                          in this macro invocation
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following trait defines an item `clone`, perhaps you need to implement it:
            candidate #1: `std::clone::Clone`
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

**Comment by kaj** [comment]
Yes, there was a change in nom 4.1.0 that made ructe break.   I havn't really figured out how to handle the change  yet, but I did release ructe 0.4.6, which locks nom at version 4.0 so it should compile.

Ok?

**Comment by kornelski** [comment]
I had `ructe = "4.0.0"` requirement in my `Cargo.toml`, and for some reason `cargo update` picks 0.4.4, not 0.4.6. Can you yank 0.4.4?
